pkg/binary/envoy/debug/pii.go

```diff
@@ -0,0 +1,72 @@
+package debug
+
+import (
+	"fmt"
+	"bitbucket.org/creachadair/shell"
+	"crypto/sha256"
+)
+
+// filter the array of logs using formatStr and validFieldNames, hash the resulting array of logs and return the final array
```

Comment:
Idiomatic method and member comments in golang start with the name of the thing being documented. So this should be `// process filters the array of logs ...`. The linter didn't complain because this is not a public method, but convention we keep private methods matching public ones.